Source/Header/DayHeaderView.swift

```diff
@@ -4,7 +4,13 @@ import DateToolsSwift
 
 public final class DayHeaderView: UIView, DaySelectorDelegate, DayViewStateUpdating, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
   public private(set) var daysInWeek = 7
-  public let calendar: Calendar
+  public var calendar: Calendar {
+    didSet {
+      daySymbolsView.calendar = calendar
+      swipeLabelView.calendar = calendar
+      configure() // TODO: Should I do it that way or better just to set a new vc without adding subview (like in state#didSet)?
```

Comment:
I think, the proper solution is to refactor the `configure()` function and extract the `addSubview` portion of it, so that it's called only when needed.
Otherwise, looks good.